src/StdUtils.sol

```diff
@@ -37,7 +42,28 @@ abstract contract StdUtils {
 
     function bound(uint256 x, uint256 min, uint256 max) internal view virtual returns (uint256 result) {
         result = _bound(x, min, max);
-        console2.log("Bound Result", result);
+        console2_log("Bound Result", result);
+    }
+
+    function bound(int256 x, int256 min, int256 max) internal view virtual returns (int256 result) {
+        require(min <= max, "StdUtils bound(int256,int256,int256): Max is less than min.");
+
+        // Shifting all int256 values to uint256 to use _bound function. The range of two types are:
+        // int256 : -(2**255) ~ (2**255 - 1)
+        // uint256:     0     ~ (2**256 - 1)
+        // So, add 2**255, INT256_MIN_ABS to the integer values.
+        //
+        // If the given integer value is -2**255, we cannot use `-uint256(-x)` because of the overflow.
+        // So, use `~uint256(x) + 1` instead.
+        uint256 _x = x < 0 ? (INT256_MIN_ABS - ~uint256(x) - 1) : (uint256(x) + INT256_MIN_ABS);
+        uint256 _min = min < 0 ? (INT256_MIN_ABS - ~uint256(min) - 1) : (uint256(min) + INT256_MIN_ABS);
+        uint256 _max = max < 0 ? (INT256_MIN_ABS - ~uint256(max) - 1) : (uint256(max) + INT256_MIN_ABS);
+
+        uint256 y = _bound(_x, _min, _max);
+
+        // To move it back to int256 value, subtract INT256_MIN_ABS at here.
+        result = y < INT256_MIN_ABS ? int256(~(INT256_MIN_ABS - y) + 1) : int256(y - INT256_MIN_ABS);
+        console2_log(string(abi.encodePacked("Bound result: ", vm.toString(result))));
```

Comment:
Instead of using `abi.encodePacked`, can just use the `log(string,string)` version. Also note the removal of the `:` for consistency since it wasn't there before
```suggestion
        console2_log("Bound result", vm.toString(result));
```

Reply:
@ZeroEkkusu feel free to merge after this change